services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java

```diff
@@ -174,13 +175,6 @@ private boolean isEfgsEnforceDateBasedDownloadAndNotSeen(String batchTag) {
       }, () -> logger.info("Batch for date {} and batchTag {} did not contain any keys.", date, batchTag));
       batchInfoService.updateStatus(batchInfo, batchContainsInvalidKeys.get() ? PROCESSED_WITH_ERROR : PROCESSED);
       return response.getNextBatchTag();
-    } catch (BatchDownloadException e) {
```

Comment:
Now I am in strong favour of having BatchDownloadException being a checked exception :-)